src/list-of-meals/mealsManager.ts

```diff
@@ -50,12 +50,34 @@ export class MealsManagerBL {
     const isMealListInLS = myLocalStorage.getMealListBL();
 
     if (isMealListInLS.length === 0) {
-      return this.getMealListBL();
+      return this.generateMealListBL();
     } else {
       return myLocalStorage.getMealListBL();
     }
   }
 
+  updateMealTime(mealList: IMealBL[], mealOrderNumEdited: number): IMealBL[] {
+    return mealList.reduce((acc: IMealBL[], item, index, _mealList) => {
```

Comment:
_mealList is not used so it should be removed to reduce confusion

Reply:
removed